### ISSUE TITLE
EnrichedUranium cost is too high

### ISSUE THREAD

**Issue by vdubikhin** [UmbraSpaceIndustries/MKS#1344]
EnrichedUranium costs 865 funds per unit, which makes it the most expensive resource. The problem comes from the fact, that EnrichedUranium can be created from ~5 Uranite(0.7 funds/unit) units. This allows to profit from EnrichedUranium production in campaign mode by simply creating a nuclear refinery near the space center.

Possible improvement is to lower EnrichedUranium cost considerably(down to 200 to put it on par with other rare resources) and 
1) either prevent player from loading Uranite in VAB/SPH
2) or increase Uranite cost to 50(50*5=250 funds, which is more than the cost of 1 EU)
to prevent player from profiting from Uranite->EnrichedUranium production without mining Uranite.

**Comment by voicey99** [comment]
The cost of EnrU acts as a balancer for how powerful the reactors are, if it were cheap then the cost barrier would be removed and the reactor stupidly overpowered. Uraninite is also pretty cheap because it's everywhere. A possible compromise would be nerfing the production rate of EnrU in the centrifuge on top of its already rather large cost.

**Comment by vdubikhin** [comment]
I agree that EnrU price should stay high as balancing factor for reactor usage. However, decreasing the production rate is not going to solve the problem. In KSP time is the resource that costs nothing: player does not pay wages to kerbals, kerbals on missions do not age or require sustenance(except for LS mods). So increasing the amount of time it takes to produce one unit of EnrU will not stop a player from abusing resource generation facilities on the launch pad.

This issue is not unique to EnrU. Even stock game allows to abuse this mechanic by producing fuel from ore. There are multiple solutions to this problem:
1) The one I initially suggested is to make all such conversions unprofitable by increasing the cost of the base resource. If the price of 5 Uranite units is higher than 1 EnrU then the only method to profit from EnrU generation is to mine Uranite.
This does not entirely solve the problem of profiting from resources without the need to leave Kerbin, but one problem at a time.

2) Increase machinery cost and/or amount of machinery needed to keep centrifuge running. This partly solves the problem as generating machinery requires complex production chain. And by the time the player can create self-sufficient colony even on Kerbin the player is likely to have enough money from contracts and rare minerals mining.

**Comment by voicey99** [comment]
There are plenty of things in KSP that only rely on a gentleman's agreement not to do. You can modify part files, you can buy ore and refine it and you can HyperEdit your crafts into space. And the only thing that's stopping you is the sense that it's not in the spirit of things to do it like that.

If someone wants to exploit the system and cheat like that, then raising Uraninite cost isn't going to stop them and it raises all sorts of other issues surrounding the raw resource being more valuable than the product. There are significant concentrations of raw resources on Kerbin for testing purposes, and RD has ruled out removing them, so they will just mine them instead, and even if they were removed they would just find another way to cheat (like stock mining and refining, for starters).

Machinery use is calculated via a universal set of balance equations, and jacking it up for the centrifuge would upset that balance. It's easier to keep everything constant.

That won't solve the problem, and all it will do is make life more difficult for the vast majority of players who don't ruthlessly exploit every system they come across.
